dissect/cobaltstrike/client.py

```diff
@@ -291,7 +292,13 @@ def run(
         self.sleeptime: int = self.bconfig.settings["SETTING_SLEEPTIME"] if sleeptime is None else sleeptime
         self.jitter: int = self.bconfig.settings["SETTING_JITTER"] if jitter is None else jitter
         self.user_agent: str = self.bconfig.settings["SETTING_USERAGENT"] if user_agent is None else user_agent
+        self.host_header: str = self.bconfig.settings["SETTING_HOST_HEADER"] if host_header is None else host_header
         self.writer = RecordWriter() if writer is not None else writer
+        _ , _ , fqdn = self.host_header.encode().partition(b": ")
+        if fqdn :
+            self.host_header = fqdn.decode().strip()
+        else:
+            self.host_header = self.domain
```

Comment:
small linting fix and if/else refactor:

```suggestion
        _, _, fqdn = self.host_header.encode().partition(b": ")
        self.host_header = fqdn.decode().strip() if fqdn else self.domain
```